notebooks/EfficientNet-example.ipynb

```diff
@@ -525,7 +525,7 @@
    "source": [
     "# The compiled module will have precision as specified by \"op_precision\".\n",
     "# Here, it will have FP32 precision.\n",
-    "trt_model_fp32 = torch_tensorrt.compile(model, inputs = [torch_tensorrt.Input((128, 3, 224, 224), dtype=torch.float32)],\n",
+    "trt_model_fp32 = torch_tensorrt.compile(model.half(), inputs = [torch_tensorrt.Input((128, 3, 224, 224), dtype=torch.float32)],\n",
```

Comment:
I dont understand this change, if everything is in FP32 why is the model being changed to fp16